examples/adjoints/adjoints_ex5/adjoints_ex5.C

```diff
@@ -271,15 +310,15 @@ void set_system_parameters(HeatSystem & system,
     {
 #ifdef LIBMESH_HAVE_PETSC
       PetscDiffSolver *solver = new PetscDiffSolver(system);
-      system.time_solver->diff_solver() = std::unique_ptr<DiffSolver>(solver);
+      system.time_solver->diff_solver() = UniquePtr<DiffSolver>(solver);
```

Comment:
What's up here?  Were you working off an older libMesh version?  This undoes a change that got rid of a use of a deprecated shim.